control-plane-operator/controllers/hostedcontrolplane/registryoperator/testdata/zz_fixture_TestReconcileServiceMonitor.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  creationTimestamp: null
+  name: cluster-image-registry-operator
+  namespace: test-namespace
+spec:
+  endpoints:
+  - bearerTokenSecret:
+      key: ""
+    interval: 60s
+    metricRelabelings:
+    - action: drop
+      regex: '*'
+      sourceLabels:
+      - __name__
+    - action: replace
+      replacement: the-cluster-id
+      targetLabel: _id
+    path: /metrics
+    relabelings:
+    - action: replace
+      replacement: the-cluster-id
+      targetLabel: _id
+    scheme: https
+    targetPort: metrics
+    tlsConfig:
+      ca:
+        secret:
```

Comment:
Just for my understanding, does this mean that whatever does the scraping needs permissions to read secrets? I think it would be good if we could avoid that

Reply:
we already require that in other service monitors (kas, kcm, etc) because we authenticate with the KAS via mtls to access the metrics endpoint.